pipelines/broad/dna_seq/germline/joint_genotyping/JointGenotyping.wdl

```diff
@@ -192,58 +196,59 @@ workflow JointGenotyping {
       output_vcf_name = callset_name + ".sites_only.vcf.gz",
       disk_size_gb = medium_disk
   }
-
-  call Tasks.IndelsVariantRecalibrator {
-    input:
-      sites_only_variant_filtered_vcf = SitesOnlyGatherVcf.output_vcf,
-      sites_only_variant_filtered_vcf_index = SitesOnlyGatherVcf.output_vcf_index,
-      recalibration_filename = callset_name + ".indels.recal",
-      tranches_filename = callset_name + ".indels.tranches",
-      recalibration_tranche_values = indel_recalibration_tranche_values,
-      recalibration_annotation_values = indel_recalibration_annotation_values,
-      mills_resource_vcf = mills_resource_vcf,
-      mills_resource_vcf_index = mills_resource_vcf_index,
-      axiomPoly_resource_vcf = axiomPoly_resource_vcf,
-      axiomPoly_resource_vcf_index = axiomPoly_resource_vcf_index,
-      dbsnp_resource_vcf = dbsnp_resource_vcf,
-      dbsnp_resource_vcf_index = dbsnp_resource_vcf_index,
-      use_allele_specific_annotations = allele_specific_annotations,
-      disk_size_gb = small_disk
-  }
-
-  if (num_gvcfs > snps_variant_recalibration_threshold) {
-    call Tasks.SNPsVariantRecalibratorCreateModel {
+  
+  if (run_vets) {
```

Comment:
Ideally, we'd have separate VQSR and VETS subworkflows, and the latter would be a direct import of the JointVcfFiltering WDL in the GATK repo. Either we'll get there in the future or we'll remove VQSR, let's see which happens first!